src/vscode-dts/vscode.d.ts

```diff
@@ -11376,8 +11397,10 @@ declare module 'vscode' {
 		 *
 		 * @param variable The variable to replace.
 		 * @param value The value to replace the variable with.
+		 * @param options Options applied to the mutator, when no options are provided this will
+		 * default to `{ applyAtProcessCreation: true }`.
```

Comment:
We should probably add a comment explaining default value of applyAtProcessCreation is actually false if any other parameter is specified, that does not seem obvious to me.

Reply:
Opps, did it this morning but forgot to push